File: Java/백준/Bronze/17356. 욱 제/욱 제.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String[] input = br.readLine().split(" ");
        int A = Integer.parseInt(input[0]);
        int B = Integer.parseInt(input[1]);

        double M = (double) (B - A) / 400;
        double result = 1 / (1 + Math.pow(10, M));

        System.out.print(result);
    }
}

File: C11/백준/Bronze/17356. 욱 제/욱 제.c
#include <stdio.h>
#include <math.h>

int main() {
    int A, B;
    scanf("%d %d", &A, &B);

    double M = (double)(B - A) / 400.0;
    double result = 1.0 / (1.0 + pow(10.0, M));

    printf("%lf", result);

    return 0;
}